modules/openapi-generator/src/main/resources/python/python-experimental/README_common.mustache

```diff
@@ -7,18 +7,20 @@ from pprint import pprint
 {{> python_doc_auth_partial}}
 # Defining host is optional and default to {{{basePath}}}
```

Comment:
Do we need to instantiate configuration here?

Reply:
I believe configuration is instantiated in the ` {{> python_doc_auth_partial}}` snippet

Reply:
actually it looks like that snippet doesn't even exit in the experimental directory...